cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,62 @@
+/// <reference types='cypress' />
+
 describe('Web Tables page', () => {
-  it('', () => {
 
+  it('working with table', () => {
```

Comment:
Inappropriate test name. Split it into different parts. You have a proper description in file readme.md, so your describe block should contain `it`s like "Add new worker", "edit worker" etc